index.html

```diff
@@ -17,10 +17,57 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container header">
+        <div class="row">
+          <div class="column-full">
+            <h1>Code Journal</h1>
+          </div>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div data-view="entry-form" class="journal-entry">
+        <form>
+          <div class="column-full">
+            <h1>New Entry</h1>
+          </div>
```

Comment:
`.column-full` won't have an impact unless it has a parent of `row`. Since `column-full` applies `flex-basis`, it will only have an impact if it's parent has a class of `row`. Create another `row` `div` and put the `column-full` inside of there